cmd/fortifyExecuteScan.go

```diff
@@ -529,25 +528,56 @@ func triggerFortifyScan(config fortifyExecuteScanOptions, command execRunner, bu
 
 		executeTemplatedCommand(command, tokenize(config.PythonInstallCommand), map[string]string{"Pip": pipVersion})
 
-		if len(config.Translate) == 0 {
-			translate := `[{"pythonPath":"`
-			translate += classpath
-			translate += ";"
-			translate += config.PythonAdditionalPath
-			translate += `","pythonIncludes":"`
-			translate += config.PythonIncludes
-			translate += `","pythonExcludes":"`
-			translate += strings.ReplaceAll(config.PythonExcludes, "-exclude ", "")
-			translate += `"}]`
-			config.Translate = translate
+		config.Translate, err = populatePipTranslate(&config, classpath)
+		if err != nil {
+			log.Entry().WithError(err).Warnf("failed to apply pythonAdditionalPath ('%s') or pythonIncludes ('%s') parameter", config.PythonAdditionalPath, config.PythonIncludes)
 		}
+
 	}
 
 	translateProject(&config, command, buildID, classpath)
 
 	scanProject(&config, command, buildID, buildLabel)
 }
 
+func populatePipTranslate(config *fortifyExecuteScanOptions, classpath string) (string, error) {
+	if len(config.Translate) > 0 {
+		return config.Translate, nil
+	}
+
+	var translateList []map[string]interface{}
+	translateList = append(translateList, make(map[string]interface{}))
+
+	translateList[0]["pythonPath"] = classpath + ";" + config.PythonAdditionalPath
+	translateList[0]["pythonIncludes"] = config.PythonIncludes
+	translateList[0]["pythonExcludes"] = strings.ReplaceAll(config.PythonExcludes, "-exclude ", "")
+
+	translateJSON, err := json.Marshal(translateList)
+
+	return string(translateJSON), err
+}
+
+func populateMavenTranslate(config *fortifyExecuteScanOptions, classpath string) (string, error) {
+	if len(config.Translate) > 0 {
+		return config.Translate, nil
+	}
+
+	var translateList []map[string]interface{}
+	translateList = append(translateList, make(map[string]interface{}))
+	translateList[0]["classpath"] = classpath
+
+	if len(config.Src) > 0 {
+		translateList[0]["src"] = config.Src
```

Comment:
It was actually not meant to provide even more configuration options as of today there are too many already. If we go for this, I would strongly vote for removing the python specific ones and resolving these via aliasing into new `src` and `exclude` parameters:
`pythonIncludes`->`src`
`pythonExcludes`->`exclude`

@fwilhe , @daniel-kurzynski , @stippi2  what's your opinion on that?

Reply:
so if a user wants both python and maven they need a pattern that fits both in the pipeline config, right?

sounds ok to me.